tmt/steps/provision/__init__.py

```diff
@@ -244,14 +250,16 @@ class Guest(tmt.utils.Common):
 
         role ....... guest role in the multihost scenario
         guest ...... name, hostname or ip address
+        is_ready ... flag to indicate guest is ready or not, which can be
+                     true or false, and it is false by default.
 
     These are by default imported into instance attributes (see the
     class attribute '_keys' below).
     """
 
     # List of supported keys
     # (used for import/export to/from attributes during load and save)
-    _keys = ['role', 'guest']
+    _keys = ['role', 'guest', 'is_ready']
```

Comment:
I think this shouldn't be accessible for load/save - it can be evaluated only in the runtime properly

Reply:
Yes, if `is_ready` is in `_keys`, it is reset to be `None` after invoking `self._guest = GuestTestcloud(data, name=self.name, parent=self.step)`
* https://github.com/teemtee/tmt/blob/main/tmt/steps/provision/testcloud.py#L280

Reply:
Since we use `@property is_ready` instead of `is_ready`, this conversation can be closed.